environments/includes/selenium.base.yml

```diff
@@ -1,27 +1,8 @@
 version: "3.5"
 services:
-  selenium-hub:
-    container_name: ${WARDEN_ENV_NAME}_selenium-hub
-    hostname: selenium-hub
-    image: selenium/hub:latest
-    healthcheck:
-      test: /opt/bin/check-grid.sh --host 0.0.0.0 --port 4444
-      interval: 20s
-      retries: 3
-    environment:
-      GRID_TIMEOUT: 0
-      GRID_BROWSER_TIMEOUT: 0
-
   selenium-chrome:
```

Comment:
```suggestion
  selenium:
```

Reply:
There are a couple places this term will need to be replaced in other files (vnc cmd, mftf docs)